packages/widgets/src/menubar.ts

```diff
@@ -772,6 +783,11 @@ export namespace MenuBar {
      */
     readonly active: boolean;
 
+    /**
+     * The index of the item in the list of items.
```

Comment:
I think this description was meant for a different variable? `focusable` is a boolean, not an index.

While we're at it, could we rename this variable? To be, focusable means "able to be focussed" but I think what this variable really means is more like "should be made focusable" (by setting `tabindex` to 0). 

What do we think about `makeFocusable` or `enableFocus` or `enableTabIndex` or...?